packages/local_auth/local_auth_darwin/darwin/local_auth_darwin.podspec

```diff
@@ -16,8 +16,10 @@ Downloaded by pub (not CocoaPods).
   s.documentation_url = 'https://pub.dev/packages/local_auth_darwin'
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
-  s.dependency 'Flutter'
+  s.ios.dependency 'Flutter'
+  s.osx.dependency 'FlutterMacOS'
   s.platform = :ios, '12.0'
```

Comment:
This should be updated to use the same format as the new line below it.

Reply:
What do you mean?

Reply:
`s.ios.deployment_target = ...`